src/main/resources/application.yml

```diff
@@ -65,6 +65,13 @@ folio:
     request:
       enabled: ${LOG_HTTP:true}
       level: FULL
+  system-user:
+    username: mod-innreach
+    password: ${SYSTEM_USER_PASSWORD:Mod-innreach-1-0-0}
+    lastname: System
+    permissionsFilePath: permissions/mod-innreach.csv
+  environment: ${ENV:folio}
+  okapi-url: ${OKAPI_URL:http://okapi:9130}
```

Comment:
Please add/remove variables from readme.md file for these env variables.

Reply:
As Environment variable is already present in readme file. I've just added okapi-url.